cf-agent/verify_databases.c

```diff
@@ -987,9 +987,8 @@ static void DeleteSQLColumns(char **name_table, char **type_table, int *size_tab
 
 /*****************************************************************************/
 
-static bool CheckSQLDataType(char *type, char *ref_type, const Promise *pp)
+static void CheckSQLDataType(char *type, char *ref_type, const Promise *pp)
 {
-    // FIXME: This function always returns true
```

Comment:
@karlhto The reason why I added this FIXME was because I didn't want to do the change you're doing here (`bool` -> `void`). As you point out in the comments above, a `Check` function should return `true` or `false`.

As it is now, this function does nothing, except for some verbose logging and maybe some segfaulting if we're lucky. I think it's probably more appropriate to return false in the cases where the type mismatch was logged.

Reply:
Please change this function to have a `bool ret` variable, returning false if one of the verbose logs were printed, and true if not.

Reply:
(`CheckSQLDataType` return value is never checked, so there is no behavior change here either way.)